pymc3/plots/posteriorplot.py

```diff
@@ -12,18 +12,28 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-try:
-    import matplotlib.pyplot as plt
-except ImportError:  # mpl is optional
```

Comment:
I don't think mpl is optional anymore - arviz is a required dependency, and mpl is a required dependency of arviz